src/BuildScriptGenerator/DefaultRunScriptGenerator.cs

```diff
@@ -65,12 +65,20 @@ public string GenerateBashScript(RunScriptGeneratorContext ctx)
                     var detectionResult = platform.Detect(ctx);
                     if (detectionResult != null)
                     {
-                        _logger.LogDebug($"Detected platform '{detectionResult.Platform}' with version '{detectionResult.PlatformVersion}'.");
+                        _logger.LogDebug($"Detected platform '{detectionResult.Platform}' with version " + 
```

Comment:
if we already had these .. why no telelmetry is there?

Reply:
I just found it in the telemetry under "GetBuildSnippets" custom events, probably have missed it. Though not found the telemtry for the runScript platform versions. Will send the screenshot to the team.